### DISCUSSION TITLE
[Players/Spells] Divine plea on lv 70 bracket

### DISCUSSION THREAD

**Discussion by Spoon722** [chromiecraft/chromiecraft#6260] [Suggestions]
Is there any chance paladins can get Divine Plea on 70 bracket?

A standard prot paladin opening rotation can nuke up to 2/3rds of their mana bar, with simple spells such as Holy Shield (10% of base mana https://www.wowhead.com/wotlk/spell=27179/holy-shield), Exorcism (8% of base mana https://www.wowhead.com/wotlk/spell=27138/exorcism), Consecration (22% of base mana https://www.wowhead.com/wotlk/spell=20924/consecration) Avenger’s Shield (26% of base mana https://www.wowhead.com/wotlk/spell=32700/avengers-shield) means a Paladin can end up using 66% of their mana on pull with no real way to cope with these issues without sacrificing threat or mitigation as the level 70 cap only allows for a few points in Benediction without negating core talents for Raiding such as Divine Sacrifice (https://www.wowhead.com/wotlk/spell=64205/divine-sacrifice) 

This isn’t that big of an issue provided you are the main tank and end up playing ping pong with your healthbar thanks to Spiritual Attunement (https://www.wowhead.com/wotlk/spell=33776/spiritual-attunement) but thanks to how most encounters are in TBC content they would get relegated to the offtank role which they can’t fulfill thanks to these mana issues without having to recur to esoteric solutions such as demonic runes + mana potions which might not be enough for prolonged fights anyways 

Level 70 bracket is gonna take a while anyway and this would be a huge QoL upgrade for any paladin that isn’t ret spec (since they get 25% of their mana per judgement anyway)

**Comment by Never6** [comment]
The thing with this is, while it may be a good idea, if you unlock later spells for one class you have to do them for everything because it would only be fair. And that is opening up a giant rabbit hole. Ferals would want Roar and Swipe, Ele would want lava burst, Hunters would want kill shot etc. While these would be good for the overall balance its not something i think they are going to do

**Comment by Amrx2** [comment]
As per the messages above, we give one class more of its toolkit early we need to give them all in order to be fair. Due to this we will not be going forward with this.
